### DISCUSSION TITLE
IPF Map

### DISCUSSION THREAD

**Discussion by antonthiel** [mtex-toolbox/mtex#903] [Ask Anything]
![legende](https://user-images.githubusercontent.com/81169124/111996360-3b958f80-8b1a-11eb-9a5e-e7e9797e2175.jpg)
its an IPF Map of Titanium

Hello Ralf, 
i have a question regarding to IPF Map. 
I dont understand why are dark areas in this map and how can i fix it? 
Is ist also possible to plot the IPF Map for another area ? ( for example: 0001, 01-10, -12-10)

**Comment by ralfHielscher** [comment]
This is not so quickly explained. Maybe you want to have a look at the paper
 [Orientations - perfectly colored](https://mtex-toolbox.github.io/publications#2016)

Second question, you can do this. Switch to 6/mmm

Ralf.
